docs/kfdrc_amplicon_architect.md

```diff
@@ -4,6 +4,40 @@ Briefly, chromosomal fragments in tumor cells break off to form circular DNA.
+ - You have `.cns` output (_not_ `.call.cns`) readily available from a previous run
+ - **OR** You have `.cnn` output from a previous run, allowing CNVkit to be run while skipping normal reference and annotation, saving time and money
+ - You have the normal alignment (bam or cram) file. In this case, CNVkit will start from scratch
+A rough run time estimate put this at about 4 hours and $1.40 in cost using spot instances on CAVATICA when CNVkit has already been run, to about 8 hours and $2.88 in cost. Run time varies if prepare step has candidates (and number of candidates) and availability of CNVkit inputs.
+### Inputs (critical):
+ - `aa_data_repo`: Reference tar ball obtained from https://datasets.genepattern.org/?prefix=data/module_support_files/AmpliconArchitect/
+ - `aa_data_ref_version`: enum, "GRCh38", "hg19", "GRCh37", "mm10", "GRCm38". Genome version in data repo to use. Should match `data_repo` downloaded.
+`tumor_align_file`: "Tumor read alignment file. Can be cram or bam", secondaryFiles: [ '^.bai?', '.crai?' ] }
+output_basename: { type: string, doc: "File name prefix for steps that require it" }
+mosek_license_file: { type: File, doc: "This tool uses some software that requires a license file. You can get a personal or institutional one from https://www.mosek.com/license/request/." }
+### Input (if cram input or .cnn not available):
+ - `reference`: Reference fasta file
+### Input (if `.cns` file available):
+ - `cnvkit_cns`: .cns file from previous CNVkit run, if available. DO NOT USE .call.cns
+### Input (if `.cnn` file available **and** `.cns` file _not_ available)
+ - `cnvkit_cnn`:If running using an existing .cnn, supply here
+### Input (if no CNVkit inputs available):
+ - `normal_align_file`: Normal read alignment file. Can be cram or bam
+ - `annotation_file`: refFlat.txt file, needed if cnv kit cnn not already built
+ - `male_input_flag`: Boolean flag indicating if input is male or not
+### Outputs:
+ - `aa_cnv_seeds`: Bed file with candidate regions to search
+ - `aa_summary`: Summary for all amplicons detected by AA
+ - `aa_cycles`: Text file for each amplicon listing the edges in the breakpoint graph, their categorization (sequence, discordant, concordant, source) and their copy counts
+ - `graph`: A text file for each amplicon listing the edges in the breakpoint graph, their categorization (sequence, discordant, concordant, source) and their copy counts
+ - `sv_pdf`: PDF image file displaying the SV view of AA
+ - `sv_png`: PNG image file displaying the SV view of AA
+ - `amplicon_classification_profiles`: Abstract classification of the amplicon
+ - `gene_list`: Genes present on amplicons with each classification
+ - `ecDNA_cts`: Not yet defined
```

Comment:
Should this be updated or will ecDNA_cts be added in a new PR?

Reply:
I will address in a new PR once our collaborators look it over